style.css

```diff
@@ -104,9 +114,12 @@ main {
 /* Header */
 
 header.box {
-  position: relative;
+  position: fixed;
```

Comment:
The header could also benefit from:

position: sticky;
top: 0;

which gets rid of the other hardcoded values.